examples/src/main/java/edu/snu/onyx/examples/beam/Broadcast.java

```diff
@@ -68,6 +68,7 @@ public void processElement(final ProcessContext c) {
         }).withSideInputs(allCollection)
     );
 
+    GenericSourceSink.write(elemCollection, outputFilePath);
```

Comment:
To check the output, 
```
final PCollection<String> result = elemCollection.apply(ParDo.of(new DoFn<String, String>() { ...
...
GenericSourceSink.write(result, outputFilePath);
```
will do the work, since there is no place to get the output of the `ParDo` applied to `elemCollection`(Thanks to @sanha).
Please check the output file after changing this. 
Great job! Thank you for the work! 👍

Reply:
@jeongyooneo I fixed it. Thank you!